source/develop/release-management/features/network/sriov-live-migration.md

```diff
@@ -0,0 +1,172 @@
+---
+title: SR-IOV Live Migration
+category: feature
+authors: amusil
+feature_name: SR-IOV Live Migration
+feature_modules: engine,network,vdsm
+feature_status: In Development
+---
+
+# SR-IOV Live Migration
+
+## Summary
+
+This feature allows you to migrate VM that is connected via SR-IOV with
+minimal network downtime and supersedes the [Live Migration Support For SRIOV][1]
+that required additional configuration inside affected VM.
+
+The state of the feature is tracked in [[RFE] [SR-IOV] Migration should not require downtime as of today][2].
+Related patches can be found on [gerrit topic:sriov_live_migration][3].
+
+## Owner
+
+*   Name: [Ales Musil](https://github.com/almusil)
+
+*   Email: <amusil@redhat.com>
+
+
+## Detailed Description
+
+In order to migrate a VM that is connected to SR-IOV passthrough profile
+we need to first unplug the SR-IOV device from a VM on the source, migrate
+the VM and plug SR-IOV device on the destination. This causes longer downtime
+during which the VM is not connected to the network.
+
+The current implementation allowed to mitigate this downtime
+[Live Migration Support For SRIOV][1], however this approach required manual
+configuration to be done on a VM which is not convenient for large setups. 
+
+In order to achieve minimal downtime the [NET_FAILOVER][4] support was added 
+to Linux kernel. The driver provides automated mechanism which is transparent 
+to the VM, and the user space. It is achieved by `failover` netdev which acts
+as master device and controls two slave devices, the `primary` device being 
+SR-IOV passtrough and the virtio as `standby`. The VM user access network
+through the `failover` device. 
+
+There is still need to provide two interfaces, the SR-IOV passtrough device
+and virtio device, to the affected VM, but the  overall configuration overhead
+has been reduced to selecting suitable vnic profile pair.
+
+
+## Prerequisites
+
+The implementation requires a [Linux kernel][5], [libvirt][6] and [qemu][7] 
+support with `net_failover`. 
+
+The [Linux kernel][5] support has to be only on guest side. 
+
+## Limitations
+
+* The virtio network has to be required to ensure connectivity on the source,
+  and the destination hosts.
+
+## Implementation
+
+### Entity Description
+
+Vm nic will get a new attribute that will point to the failover virtio 
+vnic profile. 
+
+### User Experience
+
+Upon creation of new VM nic, the user can enable failover and select 
+the failover network vnic profile. The `failover` selection is available 
+only for passthrough master vnic profiles. 
+
```

Comment:
After this section there should be image how the menu looks.

Reply:
So user will need to add 2 vNICs to the guest VM right, one virtio vNIC and one passthrough vNIC.
Can we make it clear?

Reply:
No quite the opposite. The vnic profile that will have the failover selected needs to be added to the VM as single interface and that is enough. But good point that this might be confusing I will make it clear in the feature page.

Reply:
Yes please. 
So it means only one sr-iov with failover vNIC should be added to the VM, yes?

Reply:
Exactly, to engine it is single interface.